src/pages/Authorisation.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { Link, Redirect } from "react-router-dom";
+import axios from 'axios';
+import logoImg from "../img/logo.jpg";
+import { Card, Logo, Form, Input, Button, Error } from "../components/AuthForms";
+import { useAuth } from "../context/auth";
+
+function Authorisation(props) {
+  const [isLoggedIn, setLoggedIn] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { setAuthTokens } = useAuth();
+  const referer = props.location.state.referer || '/';
```

Comment:
<img width="678" alt="Screenshot 2020-05-07 at 16 40 11" src="https://user-images.githubusercontent.com/22282622/81307972-8b45a680-9081-11ea-8cf5-6c59bd6db232.png">

Reply:
I did not have this problem - how could it appear?